### ISSUE TITLE
5.7.0 and blead@7095 make html makes man

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#2635]
Migrated from [rt.perl.org#4324](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=4324) (status was 'resolved')

Searchable as RT4324$

**Comment by p5pRT** [comment]
### From @sciurius

#### Created by @sciurius



'make html' in the pod directory makes \*\.man instead\.

In the Makefile\, the HTML= entry denotes the \.man files\.
A slight change of extension would be nice ;\-\)

Possible fix&#8203;:

\*\*\* pod/Makefile\.SH~    Sat Aug 19 17&#8203;:27&#8203;:28 2000
\-\-\- pod/Makefile\.SH    Sun Sep 17 19&#8203;:41&#8203;:35 2000
\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*
\*\*\* 22\,28 \*\*\*\*
&nbsp;  fi
&nbsp;  POD=\`echo \*\.pod\`
&nbsp;  MAN=\`echo $POD|sed 's/\\\.pod/\\\.man/g'\`
\! HTML=\`echo $POD|sed 's/perltoc\.pod//'|sed 's/\\\.pod/\\\.man/g'\`
&nbsp;  TEX=\`echo $POD|sed 's/\\\.pod/\\\.tex/g'\`
&nbsp;  
&nbsp;  echo "Extracting pod/Makefile \(with variable substitutions\)"
\-\-\- 22\,28 \-\-\-\-
&nbsp;  fi
&nbsp;  POD=\`echo \*\.pod\`
&nbsp;  MAN=\`echo $POD|sed 's/\\\.pod/\\\.man/g'\`
\! HTML=\`echo $POD|sed 's/perltoc\.pod//'|sed 's/\\\.pod/\\\.html/g'\`
&nbsp;  TEX=\`echo $POD|sed 's/\\\.pod/\\\.tex/g'\`
&nbsp;  
&nbsp;  echo "Extracting pod/Makefile \(with variable substitutions\)"

<details><summary>Perl Info</summary>

```

Flags:
    category=docs
    severity=medium

Site configuration information for perl v5.7.0:

Configured by jv at Sun Sep 17 19:13:24 CEST 2000.

Summary of my perl5 (revision 5.0 version 7 subversion 0) configuration:
  Platform:
    osname=linux, osvers=2.2.16, archname=i386-linux
    uname='linux phoenix.squirrel.nl 2.2.16 #1 mon jun 19 17:42:56 cest 2000 i686 unknown '
    config_args='-des -Darchname=i386-linux -Dprefix=/opt/perl-5.7.0'
    hint=recommended, useposix=true, d_sigaction=define
    usethreads=undef use5005threads=undef useithreads=undef usemultiplicity=undef
    useperlio=undef d_sfio=undef uselargefiles=define usesocks=undef
    use64bitint=undef use64bitall=undef uselongdouble=undef
  Compiler:
    cc='cc', ccflags ='-fno-strict-aliasing -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64', optimize='-O2', cppflags='-fno-strict-aliasing -I/usr/local/include'
    ccversion='', gccversion='egcs-2.91.66 19990314/Linux (egcs-1.1.2 release)', gccosandvers=''
    intsize=4, longsize=4, ptrsize=4, doublesize=8, byteorder=1234
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=12
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=8
    alignbytes=4, usemymalloc=n, prototype=define
  Linker and Libraries:
    ld='cc', ldflags =' -L/usr/local/lib'
    libpth=/usr/local/lib /lib /usr/lib
    libs=-lnsl -lndbm -lgdbm -ldb -ldl -lm -lc -lposix -lcrypt -lutil
    libc=/lib/libc-2.1.2.so, so=so, useshrplib=false, libperl=libperl.a
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=so, d_dlsymun=undef, ccdlflags='-rdynamic'
    cccdlflags='-fpic', lddlflags='-shared -L/usr/local/lib'

Locally applied patches:
    


@INC for perl v5.7.0:
    /opt/perl-5.7.0/lib/5.7.0/i386-linux
    /opt/perl-5.7.0/lib/5.7.0
    /opt/perl-5.7.0/lib/site_perl/5.7.0/i386-linux
    /opt/perl-5.7.0/lib/site_perl/5.7.0
    /opt/perl-5.7.0/lib/site_perl
    .


Environment for perl v5.7.0:
    HOME=/home/jv
    LANG=en_US
    LANGUAGE (unset)
    LC_ALL=en_US
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
    PATH=/opt/perl-5.7.0/bin:.:/home/jv/bin:/usr/local/bin:/bin:/usr/bin:/usr/X11R6/bin
    PERL_BADLANG (unset)
    SHELL=/bin/tcsh


```
</details>

**Comment by p5pRT** [comment]
### From The RT System itself

suggested patch supplied but not yet applied\.
